piptools/scripts/options.py

```diff
@@ -245,6 +245,7 @@ def _get_default_option(option_name: str) -> Any:
 src_files = click.argument(
     "src_files",
     nargs=-1,
+    is_eager=True,
```

Comment:
Why is this needed?

Reply:
See https://github.com/jazzband/pip-tools/issues/1902#issuecomment-1624421303. Making only `src_files` eager doesn't work though, as it won't pick up the config if not specified, then.